### PR TITLE
LPS-114703 Check segments for all AC-synced companies

### PR DESCRIPTION
**Pull request by darquesdev**
@cgoncas

### CODE REVIEW

**Comment by darquesdev** [comment]
ci:forward

File: modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsChecker.java [hunk 1]
```diff
@@ -59,16 +62,16 @@
 public class IndividualSegmentsChecker {
 
 	public void checkIndividualSegments() {
-		Optional<AsahFaroBackendClient> asahFaroBackendClientOptional =
-			_asahFaroBackendClientFactory.createAsahFaroBackendClient();
+		List<Company> companies = _companyLocalService.getCompanies();
```
Comment (cgoncas) [inline modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsChecker.java]: I think maybe is risky to get all the companies in one list, we should use a pagination search :).

Comment (darquesdev) [reply modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsChecker.java]: Good catch, not an ActionableDynamicQuery is used ;)

**Comment by darquesdev** [comment]
ci:forward

**Comment by darquesdev** [comment]
ci:forward

**Comment by darquesdev** [comment]
ci:forward

**Comment by darquesdev** [comment]
ci:forward

**Pull request closed without merging**